projects/qtaf/docs/sections/plugins/xray/Configuration.md

```diff
@@ -0,0 +1,1383 @@
+QTAF will automatically truncate parameter names and values of the iterations to the numbers provided here.
+
+!!! warning
+    If set to `#!json null`, QTAF will not truncate parameter names or values whatsoever, possibly resulting in failed result uploads.
+    Make sure to manually identify the limit your Xray instance is enforcing through the UI!
+
+***Environment variables***
+: `XRAY_RESULTS_UPLOAD_TESTS_ITERATIONS_PARAMETERS_MAX_LENGTH_NAME`
+: `XRAY_RESULTS_UPLOAD_TESTS_ITERATIONS_PARAMETERS_MAX_LENGTH_VALUE`
+
+***Type***
+: `number`
+
+***Default***
+: `#!json null`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "tests": {
+              "iterations": {
+                "parameters": {
+                  "maxLength": {
+                    "name": 42,
+                    "value": 50
+                  }
+                }
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_TESTS_ITERATIONS_PARAMETERS_MAX_LENGTH_NAME=42
+        XRAY_RESULTS_UPLOAD_TESTS_ITERATIONS_PARAMETERS_MAX_LENGTH_VALUE=50
+        ```
+
+<hr/>
+
+#### `info`
+
+Settings regarding general test case information.
+
+##### `keepJiraSummary`
+
+When uploading test results, QTAF can automatically rename existing test issues to the test case's name defined within QTAF.
+This setting toggles whether this is allowed to happen or whether the test case's existing summaries should be kept.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_TESTS_INFO_KEEPJIRASUMMARY`
+
+***Type***
+: `boolean`
+: Accepted strings (case-insensitive):
+
+    - `#!json "true"`
+      `#!json "1"`
+      `#!json "y"`
+
+    - `#!json "false"`
+      `#!json "0"`
+      `#!json "n"`
+
+***Default***
+: `#!json false`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "tests": {
+              "info": {
+                "keepJiraSummary": true
+              }
+            }
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_TESTS_INFO_KEEPJIRASUMMARY="y"
+        ```
+
+<hr/>
+
+##### `steps`
+
+When uploading test results, QTAF can automatically update or change the steps defined within existing test issues with the steps it executed itself.
+These settings control how and whether this is allowed to happen.
+
+###### `update`
+
+Toggles whether QTAF is allowed to update existing test issue steps in Xray.
+Setting this to `#!json true` will result in QTAF replacing _all_ existing steps with the ones it executed itself.
+To not modify any existing steps, set this to `#!json false`.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_TESTS_STEPS_UPDATE`
```

Comment:
It should be TESTS_INFO_ instead of TESTS_
This error occurs multiple times in the document

Reply:
Fixed in 5cd773c20aa16236742c2163275384967f69bf99.